group_project_v2/templates/html/stages/dashboard_view.html

```diff
@@ -17,7 +17,14 @@
     {% for stat_label, stat_value in stats.items %}
       <tr class="group-project-stage-stat">
         <td class="group-project-stage-stat-label">{{ stat_label }}</td>
-        <td class="group-project-stage-stat-value">{{ stat_value|floatformat:"0" }}%</td>
+        <td class="group-project-stage-stat-value">
```

Comment:
@jbzdak nit: please add `text-align: right` to this class - I believe it right alignment would give it a slightly better look (both to empty set and percent vlaues).